force-app/main/default/classes/LeadTriggerHandlerTest.cls

```diff
@@ -0,0 +1,190 @@
+        leads.add(new Lead(FirstName = 'Oxana', LastName = 'Suvorova', Company = 'Suvorova Schools', Title = 'manage', Industry='Technology', LeadSource='Purchased List', Email='o.suvorova@schools.com'));
+        leads.add(new Lead(FirstName = 'John', LastName = 'George', Company = 'George Elite', Title = 'head of department', Industry='Apparel', LeadSource='Purchased List', Email='j.george@elite.com'));
+        leads.add(new Lead(FirstName = 'Navid', LastName = 'Mudassir', Company = 'M.U.D.', Title = 'exec', Industry='Technology', LeadSource='Web', Email='n.mudassir@mud.com'));
+        leads.add(new Lead(FirstName = 'Raghavendra', LastName = 'Yadavalli', Company = 'R.Y.', Title = 'chief', Industry='Utilities', LeadSource='Purchased List', Email='r.yadavalli@ry.com'));
+        leads.add(new Lead(FirstName = 'Tony', LastName = 'Kirumba', Company = 'Tony\'s Turtles', Title = 'head', Industry='Retail', LeadSource='Purchased List', Email='t.kirumba@turtlesrule.com'));
+        leads.add(new Lead(FirstName = 'Ariel', LastName = 'T.', Company = 'Under the Sea', Title = 'assist', Industry='Hospitality', LeadSource='Web', Email='a.t@underwater.com', Phone='654-321-5823'));
+        leads.add(new Lead(FirstName = 'Dallas', LastName = 'Bailey', Company = 'Bailey Tea', Title = 'deputy', Industry='Retail', LeadSource='Phone Inquiry', Email='d.bailey@teatime.com'));
+        leads.add(new Lead(FirstName = 'Tyler', LastName = 'Riden', Company = 'Rock n Riden', Title = 'jr', Industry='Construction', LeadSource='Partner Referral', Email='t.riden@rock4life.com'));
+        leads.add(new Lead(FirstName = 'Riley', LastName = 'Meyer', Company = 'Developer Futures', Title = 'developer', Industry='Technology', LeadSource='Web', Email='r.meyer@dev.com', Phone='987-654-3210'));
+
+        Database.insert(leads, AccessLevel.SYSTEM_MODE);
+    }
+
+    @IsTest
+    static void testHandleTitleNormalization() {
+
+        // Check Results
+        List<Lead> titleUpdates = new List<Lead>([SELECT Id, FirstName, LastName, Company, Title
+                                                    FROM Lead]);
+        List<Lead> vicePresidents = new List<Lead>();
+        List<Lead> managers = new List<Lead>();
+        List<Lead> executives = new List<Lead>();
+        List<Lead> assistants = new List<Lead>();
+        List<Lead> otherTitles = new List<Lead>();
+        for (Lead ld : titleUpdates) {
+            if (ld.Title == 'Vice President') {
+                vicePresidents.add(ld);
+            } else if (ld.Title == 'Manager') {
+                managers.add(ld);
+            } else if (ld.Title == 'Executive') {
+                executives.add(ld);
+            } else if (ld.Title == 'Assistant') {
+                assistants.add(ld);
+            } else {
+                otherTitles.add(ld);
+            }
+        }
+
+        System.assertEquals(3, vicePresidents.size(), 'A Lead Title was not correctly set to Vice President');
```

Comment:
It's now recommended to use the assert class for test assertions https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_class_System_Assert.htm